datasets/us/dod_chinese_milcorps/us_dod_chinese_milcorps.yaml

```
@@ -0,0 +1,21 @@
+name: us_dod_chinese_milcorps
+title: "Chinese military companies"
+url: https://www.defense.gov/News/Releases/Release/Article/3661985/dod-releases-list-of-peoples-republic-of-china-prc-military-companies-in-accord/
+
```

https://zavod.opensanctions.org/metadata/#data-coverage

For a snapshot like this, I think you can set start and end date to when the source was published - 2024-01-31 - or if that's unclear, the date we added it / put it in docs

```suggestion
coverage:
  frequency: never
  start: 2024-01-31
  end: 2024-01-31
```